x-pack/plugins/security_solution/scripts/endpoint/api_emulator/emulator_plugins/crowdstrike/routes/batch_rtr_command_route.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { buildCrowdstrikeRoutePath, TEST_AGENT_ID, TEST_SESSION_ID } from './utils';
+import type { ExternalEdrServerEmulatorRouteHandlerMethod } from '../../../external_edr_server_emulator.types';
+import type { EmulatorServerRouteDefinition } from '../../../lib/emulator_server.types';
+
+export const batchRTRCommandRoute = (): EmulatorServerRouteDefinition => {
+  return {
+    // we use `combined` api - which returns just one complete response, otherwise it would be coming in batches
+    path: buildCrowdstrikeRoutePath('/real-time-response/combined/batch-command/v1'),
+    method: 'POST',
+    handler: batchRTRCommandSuccessHandler,
+  };
+};
+
+// @ts-expect-error - example of missing file error
```

Comment:
I see a few of these TS ignores. Whats the error being generated?

Reply:
true, I marked them as expected - because these constant are not being used. I added them for the reference + we might start returning responses with errors at some point. For development it is enough to switch the handler.